pkg/internal/apply/layerApplier.go

```diff
@@ -175,6 +205,7 @@ func (a KubectlLayerApplier) Apply(layer layers.Layer) (err error) {
 	}
 
 	output, err := a.kubectl.Apply(sourceDir).WithLogger(layer.GetLogger()).Run()
+	//output, err := a.kubectl.Apply(sourceDir).WithLogger(layer.GetLogger()).DryRun()
```

Comment:
`layer.GetLogger()` shouldn't be needed. I'd say a logger isn't part of a layer and is an implementation detail that should be private. WDYT?